crossdock/endtoend/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/uber/jaeger-client-go/crossdock/endtoend/server"
+)
+
+const (
+	defaultPort = ":8000"
+)
+
+func main() {
+	handler := &server.Handler{}
+
+	http.HandleFunc("/init", handler.Init)
+	http.HandleFunc("/trace", handler.Trace)
+	http.ListenAndServe(defaultPort, nil)
```

Comment:
I would strongly prefer if we integrate this new behavior into the existing crossdock server, and build out a single docker image.